nv_dds.h

```diff
@@ -4,13 +4,17 @@
 #include <deque>
 #include <istream>
 
-#include <assert.h>
-#include <stdint.h>
+#include <cassert>
+#include <cstdint>
 
-#ifdef __APPLE__
-#include <OpenGL/gl.h>
-#else
-#include <GL/gl.h>
+#define NV_DDS_GL_SUPPORT 1
+
+#if NV_DDS_GL_SUPPORT == 1

```

Comment:
`#ifndef NV_DDS_NO_GL_SUPPORT`. this allows to just define it on commandline without modifying the source